### ISSUE TITLE
Easter sunday is holiday in VIC, QLD, ACT in AU

### ISSUE THREAD

**Issue by isha1111** [dr-prodigy/python-holidays#159]
in file python.py , line 1871 

```
if self.prov == 'NSW':
            self[easter(year)] = "Easter Sunday"

```
Can you please ACT, VIC, QLD states as well. As they also consider easter sunday as public holiday. Thanks

**Comment by dr-prodigy** [comment]
Hi,
if possible, I would kindly ask you to implement the changes and tests, and pack them in a PR: I'll surely be merging it in the next release.

Thank you very much, cheers

**Comment by dr-prodigy** [comment]
Fixed in 64d2366fe3226cf50366f2a13004f0588702258f
